.github/actions/alembic-install-dep/action.yml

```diff
@@ -5,26 +5,33 @@ inputs:
     description: 'CPU architecture to build for'
     required: false
     default: 'x86_64'
-    
+  version:
+    description: 'Version of alembic to build'
+    required: true
+
 runs:
   using: "composite"
   steps:
 
-    # Alembic is part of VFX reference platform (CY2025: 1.8.x)
+    - name: Check required inputs
```

Comment:
is it needed? There's a `required` flag on inputs

Reply:
required as no effect: https://stackoverflow.com/questions/68804484/why-are-required-inputs-to-github-actions-not-enforced